libs/models/src/lib/app/stateModel.ts

```diff
@@ -0,0 +1,19 @@
+import { AppErrorLogs } from './errorLog';
```

Comment:
use `kebab-case` for files unless they:
1. define and export a react component (Eg: 'HomeScreen.tsx`)
2. define and export a single class (Eg: `HttpService.ts`)

Reply:
Changed file names in 47ae59b714a45c2b1d3692f62afe5c19b4cdf1b3